src/Core/MailTemplate/ThemeCollection.php

```diff
@@ -40,4 +40,21 @@ protected function getType()
     {
         return ThemeInterface::class;
     }
+
+    /**
+     * @param string $themeName
+     *
+     * @return ThemeInterface|null
+     */
+    public function getByName($themeName)
+    {
+        /** @var ThemeInterface $theme */
+        foreach ($this as $theme) {
+            if ($themeName === $theme->getName()) {
```

Comment:
use https://www.doctrine-project.org/api/collections/latest/Doctrine/Common/Collections/ArrayCollection.html#method_filter instead ?

Reply:
Oh I didn't think we could benefit from this method, great idea!
But in this case it's not relevant, filter returns a collection and we only want one item returned in this method

Reply:
Combine methods:

```php
$this->filter($filterCondition)->first();
```

Reply:
Ok, done!